### DISCUSSION TITLE
Centrify support for MFA in Cockpit

### DISCUSSION THREAD

**Discussion by Sangeeta-Mahajan-Imprivata** [cockpit-project/cockpit#18891] [General]
Hello,
  Does Cockpit support MFA using Centrify (AD authentication) on RHEL 8?

Thanks,
sangeeta

**Comment by martinpitt** [comment]
It does support standard kerberos authentication against AD, see https://cockpit-project.org/guide/latest/sso.html . I haven't heard of "Centrify" -- anything that happens in PAM should more or less work, if it requires custom code in the authentication client (i.e. Cockpit), it most likely won't.
